docs/datamodel/constraints.rst

```diff
@@ -220,3 +220,5 @@ using arbitrary EdgeQL expressions. The example below uses the built-in
   * - :ref:`DDL > Constraints <ref_eql_ddl_constraints>`
   * - :ref:`Introspection > Constraints <ref_eql_introspection_constraints>`
   * - :ref:`Standard Library > Constraints <ref_std_constraints>`
+  * - `Tutorial > Advanced EdgeQL > Constraints
+      <https://www.edgedb.com/tutorial/advanced-edgeql/constraints>`_
```

Comment:
@1st1 @jaclarke do we have some intersphinx capability to be able to use refs here to avoid hardcoding the URLs?

Reply:
@elprans I almost asked about this, but it didn't look like the tutorial is built in Sphinx so I assumed it isn't possible.

Reply:
@elprans These links are relative now, and I've added https://github.com/edgedb/website/issues/5 to address reference linking into the tutorial. For some reason, I had it in my head that we couldn't do relative links in Sphix when I was putting this together, so thanks for calling that out.